backend/slack_commands/commands/list_teams.py

```
@@ -0,0 +1,23 @@
+"""List teams command — shows teams the user belongs to."""
+import requests
+
+from global_utils.identity_client import IdentityClient
+from slack_commands.commands.base import CommandHandler
+from slack_commands.formatters import format_team_list
+from slack_commands.models import SlackCommand, SlackResponse
+
+
+class ListTeamsCommand(CommandHandler):
+
+    def __init__(self, identity_client: IdentityClient):
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)